### PR TITLE
Style Guide and Glossary cross-reference edits (A-C)

### PR DESCRIPTION
**Pull request by rachel-feinberg**
# Pull Request/Issue Resolution

### CODE REVIEW

File: _docs/_api/endpoints/templates/content_blocks_templates/post_update_content_block.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Authorization: Bearer YOUR-REST-API-KEY
 | `content_block_id`|	Required |	String | Your content block's API identifier.|
 | `name` | Optional | String | Name of the content block. Must be less than 100 characters. |
 | `description` | Optional | String | Description of the content block. Must be less than 250 characters. |
-| `content` | Optional | String | HTML or text content within content blocks.
+| `content` | Optional | String | HTML or text content within Content Blocks.
```
Comment (KellieHawks) [inline _docs/_api/endpoints/templates/content_blocks_templates/post_update_content_block.md:54]:
```suggestion
| `content` | Optional | String | HTML or text content within Content Blocks. |
```

File: _docs/_hidden/other/dp-technical-assistance.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ Warning! Don't make any changes to this document without approval from the legal
 
 # Data Protection Technical Assistance
 
-There are a range of data protection laws that regulate what organizations can do with personal data (“**Data Protection Laws**”), including the EU and UK General Data Protection Regulation (“**GDPR**”), the California Consumer Privacy Act as amended by the California Privacy Rights Act (“**CCPA**”) and the Health Insurance Portability and Accountability Act (“**HIPAA**”). There are other national, state, and industry-specific Data Protection Laws and regulations that may apply to your business.
+There are a range of data protection laws that regulate what organizations can do with personal data (“**Data Protection Laws**”), including the EU and UK General Data Protection Regulation (“**GDPR**”), the  as amended by the California Privacy Rights Act (“**CCPA**”) and the Health Insurance Portability and Accountability Act (“**HIPAA**”). There are other national, state, and industry-specific Data Protection Laws and regulations that may apply to your business.
```
Comment (KellieHawks) [inline _docs/_hidden/other/dp-technical-assistance.md:16]: CCPA or CPRA?

Comment (rachel-feinberg) [reply _docs/_hidden/other/dp-technical-assistance.md:16]: I think it's meant to be "the California Consumer Privacy Act ("**CCPA**") as amended by the California Privacy Rights Act (“**CCPA**”)"

However, I just saw the note "@@ -13,7 +13,7 @@ Warning! Don't make any changes to this document without approval from the legal" so maybe this shouldn't be edited at all?

Comment (KellieHawks) [inline _docs/_hidden/other/dp-technical-assistance.md:16]:
```suggestion
There are a range of data protection laws that regulate what organizations can do with personal data (“**Data Protection Laws**”), including the EU and UK General Data Protection Regulation (“**GDPR**”), the California Consumer Privacy Act  (“**CCPA**”) and the Health Insurance Portability and Accountability Act (“**HIPAA**”). There are other national, state, and industry-specific Data Protection Laws and regulations that may apply to your business.
```
```suggestion
There are a range of data protection laws that regulate what organizations can do with personal data (“**Data Protection Laws**”), including the EU and UK General Data Protection Regulation (“**GDPR**”), the California Consumer Privacy Act (“**CCPA**”) and the Health Insurance Portability and Accountability Act (“**HIPAA**”). There are other national, state, and industry-specific Data Protection Laws and regulations that may apply to your business.
```

Comment (KellieHawks) [inline _docs/_hidden/other/dp-technical-assistance.md:16]:
```suggestion
There are a range of data protection laws that regulate what organizations can do with personal data (“**Data Protection Laws**”), including the EU and UK General Data Protection Regulation (“**GDPR**”), the California Consumer Privacy Act (“**CCPA**”) and the Health Insurance Portability and Accountability Act (“**HIPAA**”). There are other national, state, and industry-specific Data Protection Laws and regulations that may apply to your business.
```

**Comment by rachel-feinberg** [comment]
@KellieHawks I made a few edits for capitalization. Hit merge conflicts but I think I resolved them. This PR is ready for full consideration now (phew) Sorry for any hiccups

**Review by KellieHawks — APPROVED** [review]

**Pull request merged**